connector/oidc/oidc.go

```diff
@@ -39,6 +39,9 @@ type Config struct {
 
 	// Override the value of email_verifed to true in the returned claims
 	InsecureSkipEmailVerified bool `json:"insecureSkipEmailVerified"`
+
+	// Configurable key which contains the user id claims
```

Comment:
[nit] It's just one claim, isn't it? 
```suggestion
	// Configurable key which contains the user id claim
```

Reply:
Thanks for your suggestion 👍 https://github.com/dexidp/dex/pull/1448/commits/5e68d43af2aeb4802105937f9102bfcd52ff8f24